paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExperimentalLayoutApi.kt

```diff
@@ -0,0 +1,13 @@
+package com.stripe.android.paymentsheet
+
+import androidx.annotation.RestrictTo
+
+@RequiresOptIn(message = "Layout support is beta. It may be changed in the future without notice.")
+@Retention(AnnotationRetention.BINARY)
+@Target(
+    AnnotationTarget.CLASS,
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY
+)
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+annotation class ExperimentalLayoutApi
```

Comment:
Should this be `ExperimentalPaymentSheetLayoutApi`?

Reply:
Works for me. Let me update the API review too.